packages/salesforcedx-vscode-core/src/commands/forceSourcePull.ts

```diff
@@ -68,9 +58,8 @@ const workspaceChecker = new SfdxWorkspaceChecker();
 const parameterGatherer = new EmptyParametersGatherer();
 
 export async function forceSourcePull(this: FlagParameter<string>) {
-  const { flag, commandVersion } = this || {};
-  const command =
-    commandVersion === CommandVersion.Legacy ? pullCommandLegacy : pullCommand;
+  const { flag } = this || {};
+  const command = pullCommand;
```

Comment:
I think we can collapse this further and strike line 62, just passing `pullCommand` into `ForceSourcePullExecutor`